.github/ISSUE_TEMPLATE/simeng-bug-or-issue-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: SimEng Bug or Issue Report
+about: Report a bug or un-expected SimEng behaviour
```

Comment:
For small but obvious issues such as NYI instruction this is probably more work for us and the submitter than is necessary. Should there be a bypass for using this template?? Just a thought

Reply:
We could add a second template for NYI or Alias exceptions, as I agree this is quite verbose for these cases

Reply:
I think that would be sensible if github allows for it

Reply:
Added NYI etc Issue Template